src/users/users.module.ts

```diff
@@ -41,7 +42,8 @@ import { RpxTranslationModule } from 'rpx-xui-translation';
   exports: [...fromContainers.containers, ...fromComponents.components],
   declarations: [
     ...fromContainers.containers,
-    ...fromComponents.components
+    ...fromComponents.components,
+    ManageUserFailureComponent
```

Comment:
Add `ManageUserFailureComponent` to the `...fromComponents.containers` like the other top level components